gh_actions_exporter/metrics.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, settings: Settings):
             "job_name",
             "repository_visibility",
             "runner_type",
+            "workflow_name",
```

Comment:
Check if it's not too much work to add a `self.common_labelnames` var with:
* repository
* workflow_name
* repository_visibility

Reply:
Done! Tell me if it's good now